src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculator.kt

```diff
@@ -12,48 +12,40 @@ class ChangeLevelCalculator(
 ) {
 
   fun calculateChangeLevel(crn: String): TierLevel<ChangeLevel> {
-    if (shouldCalculateChangeLevel(crn)) {
-
-      val points = getOasysNeedsPoints(crn) + getOgrsPoints(crn) + getIomNominal(crn)
-      val tier = calculateTier(points)
-
-      return TierLevel(tier, points)
-    }
-    return TierLevel(ChangeLevel.ZERO, 0)
-  }
-
-  private fun getIomNominal(crn: String): Int {
-    // We don't care about the full list, only if there is IOM Nominal
-    return if (communityApiDataService.getComplexityFactors(crn).any { it == ComplexityFactor.IOM_NOMINAL }) 2 else 0
+    return if (hasMandateForChange(crn)) {
+      val totalPoints = getOasysNeedsPoints(crn) + getOgrsPoints(crn) + getIomNominalPoints(crn)
+      val tier = when {
+        totalPoints >= 20 -> ChangeLevel.THREE
+        totalPoints in 10..19 -> ChangeLevel.TWO
+        else -> ChangeLevel.ONE
+      }
+      TierLevel(tier, totalPoints)
+    } else TierLevel(ChangeLevel.ZERO, 0)
   }
 
-  private fun shouldCalculateChangeLevel(crn: String): Boolean =
+  private fun hasMandateForChange(crn: String): Boolean =
     when {
-      communityApiDataService.isCurrentCustodialSentence(crn) -> true
-      communityApiDataService.isCurrentNonCustodialSentence(crn) -> hasNoUnpaidWorkOrRestrictiveRequirements(crn)
+      communityApiDataService.hasCurrentCustodialSentence(crn) -> true
```

Comment:
Happy to change back, but I like to use has if we're checking a collection. i.e 'has any current custodial sentence'